astropy/nddata/mixins/ndslicing.py

```diff
@@ -116,8 +118,6 @@ def _slice_mask(self, item):
     def _slice_wcs(self, item):
         if self.wcs is None:
             return None
-        try:
-            return self.wcs[item]
-        except TypeError:
-            log.info("wcs cannot be sliced.")
-        return self.wcs
+
+        llwcs = SlicedLowLevelWCS(self.wcs.low_level_wcs, item)
+        return HighLevelWCSWrapper(llwcs)
```

Comment:
What about adding ``SlicedHighLevelWCS`` which basically does this?

Reply:
That seems like a little bit of overkill, certainly at the moment given how we are still testing out sllwcs with gWCS etc.